far/vmenu.cpp

```diff
@@ -172,6 +171,47 @@ int find_nearest(std::ranges::contiguous_range auto const& Range, const int Pos,
 			: FindPos(drop(Pos), take(Pos) | reverse));
 }
 
+template <std::integral T>
```

Comment:
Well, we can specialize them for the `std::pair<int, int>` which is the only one required, and for `std::vector<int>& Markup`, though I am not sure it makes things really easier to understand. What we cannot do is to avoid generic `std::ranges::input_range auto const& Slices` because we actually call this function with different view types. I will apply the changes.